styleguide/git.md

```diff
@@ -11,10 +11,9 @@ Joinbox uses a more specific version of the git-flow branching model
 - feature branches may only merged into the develop branch if they are tested and work correctly
 - for each iteration (release cycle, milestone) a new release branch is created from develop as soon everything is ready.
 - the release branch is published to a testing server for acceptance tests by the client. the client should receive the change log of the release branch
-- fixes for the release branch are merged into the release branch
-- the release branch may be merged into develop or another release branch if it is required
-- hot fixes are only created if they must be applied to the master branch for immediate release. they are based on the master branch
-- after a hot fix was released the master branch is merged into the develop branch
+- fixes for the release branch are merged into the release branch. Branches are bases on the HEAD of the release branch
```

Comment:
i don't think it has to be the head of the master, maybe one has to apply a hotfix to older code that is running on some system.

Reply:
It's about fixes for the release branch not hotfixes for the master

Reply:
`Branches are bases on the HEAD of the release branch`. 

- Which branches are based on the head of the release branch?
- there is a typo: «bases»